pnpm-lock.yaml

```diff
@@ -18246,7 +18188,7 @@ packages:
     dependencies:
       ansi-styles: 3.2.1
       chalk: 2.4.2
-      cross-spawn: 7.0.5
+      cross-spawn: 6.0.6
```

Comment:
Just wondering about this downgrade? @donal-tobin-sap

Reply:
Good catch, I had a typo, the fix is the upgrade to npm-run-all2 which is also waiting in renovate.